pinot-broker/src/main/java/com/linkedin/pinot/broker/broker/BrokerServerBuilder.java

```diff
@@ -244,7 +251,11 @@ public void start() throws Exception {
 
   public void stop() throws Exception {
     LOGGER.info("Shutting down Network !!");
-
+    try {
+      Thread.sleep(delayedShutdownTimeMs);
+    } catch (Exception e) {
```

Comment:
Change this to InterruptedException, change log message to "Interrupted while waiting for shutdown delay period of {} ms."